travvy/src/App.vue

```diff
@@ -160,8 +160,8 @@ border-radius: 10px;
 }
 
 .info {
- background-color: #89C4A0;
-  color: white;
+ background-color: #C7EEA9;
+  color: black;
```

Comment:
Thanks for fixing this! Hopefully the users will be able to read the text better now.

Reply:
agreed, thats good implementation from the feedback the TAs gave us